### ISSUE TITLE
Remove console.log in addons.addons.setConfig

### ISSUE THREAD

**Issue by sivaraj-v** [storybookjs/storybook#14613]
**Describe the bug**
Remove printing console.log in storybook 

![image](https://user-images.githubusercontent.com/6199252/114835258-01ed3700-9def-11eb-814e-44096daf9d31.png)

```
    "@storybook/addon-a11y": "^6.2.7",
    "@storybook/addon-actions": "6.2.7",
    "@storybook/addon-docs": "6.2.7",
    "@storybook/addon-jest": "6.2.7",
    "@storybook/addon-knobs": "6.2.7",
    "@storybook/addon-links": "6.2.7",
    "@storybook/addon-viewport": "6.2.7",
    "@storybook/react": "6.2.7",
    "@whitespace/storybook-addon-html": "5.0.0",
    
```

**Comment by shilman** [comment]
Do you have a repro repo you can share?

**Comment by sivaraj-v** [comment]
Issue resolved in 6.2.9, we can close this issue
